react/react-basics/step4.md

```diff
@@ -1,68 +1,19 @@
-Now we'll add content to `CardHead`, `CardActions` and `CardHeader`.
+Add content to the `CardBody` of the card.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Add a text component to the `CardHeader` component
+2) <strong>Locate the `<CardBody>` component.</strong>
 
-The text component can wrap any static HTML content you want to place on your page to provide correct formatting when using standard HTML tags. It comes in several variations, such as `h1`, `p`, `a`, `small`, `blockquote`, and `pre`.
+It will look like the following:
 
-In this exercise, we'll be using the `p` variation, which is specified with `component={TextVariants.p}`.
-
-Add the following code to the `CardHeader` component:
-
-<pre class="file" data-target="clipboard">
-&lt;CardHeader&gt;
-  &lt;Text component={TextVariants.p}&gt;
-    PatternFly
-  &lt;/Text&gt;
-&lt;/CardHeader&gt;
-</pre>
-
-
-2) Add a brand to the `CardHead` component
-
-The brand component is used to place a product logotype on a screen. All you need to provide are the source image or svg and alt text.
-
-<pre class="file" data-target="clipboard">
-&lt;CardHead&gt;
-  &lt;Brand src=&quot;https://pf4.patternfly.org/assets/images/pf-logo-small.svg&quot; alt=&quot;Patternfly Logo&quot; /&gt;
-  &lt;CardActions&gt;
-  &lt;/CardActions&gt;
-&lt;/CardHead&gt;
+<pre class="file">
+&lt;CardBody&gt;
+&lt;/CardBody&gt;
 </pre>
 
-3) Add a plain close button to the card component in the `CardActions` component
-
-Buttons communicate and trigger actions a user can take in an application or website. They come in several variations, such as `primary`, `secondary`, `danger`, `plain`, and `link`.
-
-PatternFly React provides a variety of icons as well as components, layouts, charts, and demos. They're just as easy to use and they compile into svgs. You can use them inside buttons and other components.
-
-We'll be using a button with the `plain` variant and an icon.
-
-<pre class="file" data-target="clipboard">
-&lt;CardActions&gt;
-  &lt;Button variant=&quot;plain&quot;&gt;
-    &lt;TimesIcon /&gt;
-  &lt;/Button&gt;
-&lt;/CardActions&gt;
-</pre>
-
-4) Add a subhead below the "PatternFly" Text component in the `CardHeader` component
-
-TextContent is used to wrap Text components and provides text with extra spacing and styling.
+3) <strong>Add content inside the `<CardBody>`.</strong>
 
 <pre class="file" data-target="clipboard">
-&lt;CardHeader&gt;
-  &lt;Text component={TextVariants.p}&gt;
-    PatternFly
-  &lt;/Text&gt;
-  &lt;TextContent&gt;
-    &lt;Text component={TextVariants.small}&gt;
-      Provided by Red Hat
-    &lt;/Text&gt;
-  &lt;/TextContent&gt;
-&lt;/CardHeader&gt;
+  PatternFly is an open source design system created to enable consistency and usability across a wide range of applications and use cases. PatternFly provides clear standards, guidance, and tools that help designers and developers work together more efficiently and build better user experiences.

```

Comment:
```suggestion
 <strong>Note:</strong>a PatternFly is an open-source design system created to enable consistency and usability across a wide range of applications and use cases. PatternFly provides clear standards, guidance, and tools that help designers and developers work together more efficiently and build better user experiences.
```

Reply:
This is actually content that sits inside of the card component, so it shouldn't be in a note